src/device-registry/utils/create-event.js

```diff
@@ -18,7 +18,6 @@ const { getDevicesCount, list, decryptKey } = require("./create-monitor");
 const HTTPStatus = require("http-status");
 const redis = require("../config/redis");
 const axios = require("axios");
-const { kafkaConsumer } = require("../config/kafkajs");
```

Comment:
Thanks @BenjaminSsempala , could you add some guiding PR review comments about this deletion?